content/platform/alert-runbooks.md

```diff
@@ -164,3 +164,45 @@ Some current outbound requests are affected.
 ##### Diagnosis & Mitigation
 
 Follow [NAT Gateway IP Allocation Failures](./troubleshooting#nat-gateway-ip-allocation-failures) section.
+
+
+#### ArgoCDApplicationIsNotHealthy
+
+##### Meaning
+
+One of the application components is not healthy, resulting in the whole application being marked as unhealthy.
+
+##### Impact 
+
+Tenant resources are allocated as the ArgoCD applications,
+meaning that part of the tenant functionality might not be working as expected.
+For example, some of the tenant permissions might be missing.
+
+##### Diagnosis & Mitigation
+
+1. Identify the problematic component by checking the ArgoCD UI: https://argocd.{{ internalServices.domain }}
```

Comment:
is internalservices.domain a variable? will get replaced based on env?

Reply:
it's a variable from `dplatform-environments` repo.
It won't be replaced, since we deploy only single instance of docs, but we use it everywhere in docs to describe  variables